api/context.go

```diff
@@ -185,6 +185,20 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	// TEMPORARY CODE FOR 3.9, REMOVE FOR 3.10
```

Comment:
Can this be removed for 3.10? What if someone upgrades from 3.8 directly to 3.10?